### DISCUSSION TITLE
Using Multiple Mice

### DISCUSSION THREAD

**Discussion by Matt-B-Anderson** [nut-tree/nut.js#436] [General]
I have been intrigued by the idea of being able to record input from a mouse and play it back while also being able to do other things on my same computer. I have found a free app called eithermouse that allows you to have two cursors at once. With the current project am I able to differentiate between the two and pick a mouse to run my scripts with?

**Comment by s1hofmann** [comment]
So far I've never heard of `eithermouse`, thus I don't know how it manages multiple cursors.

My gut feeling tells me that this is more dependent on the software actually managing multiple cursors.
With nut.js, however, there's nothing implemented to add some specific identifier whatsoever to input events, so I'd say no, this is not possible.
